src/main/webapp/app/assessment/assessment-detail/assessment-detail.component.ts

```diff
@@ -42,4 +42,17 @@ export class AssessmentDetailComponent {
     public get sanitizedText(): string {
         return sanitize(this.text);
     }
+    allowDrop(event: DragEvent) {
+        event.preventDefault();
+    }
+    drop(event: any) {
```

Comment:
please add a JSDoc comment here, that this method connects the GradingInstruction with the Feedback
link/reference the corresponding drag method in the JSDoc